src/MOI_wrapper.jl

```diff
@@ -442,8 +446,12 @@ MOI.supports(::Optimizer, ::MOI.Silent) = true
 MOI.get(model::Optimizer, ::MOI.Silent) = model.silent
 
 function MOI.set(model::Optimizer, ::MOI.Silent, flag::Bool)
+    if flag
+        Highs_runQuiet(model)
```

Comment:
`Highs_runQuiet` is just setting the mode as silent or also running the model?

Reply:
It just does what's necessary so that HiGHS will run quietly when Highs_run() is next called, but I see what you mean!

Reply:
ok thanks for the precision. Would it be worth it to rename it as `Highs_setQuiet()` in HiGHS? (With a deprecation mechanism to avoid being breaking)

Reply:
Sounds like a better name now that you've raised the issue of confusion.

Reply:
I added setHighsLogfile and setHighsOutput to the C API, so I don't think we really need runQuiet.

Reply:
Indeed, although it remains for back-compatibility. Highs_setQuiet() is now unnecessary - unless there are users confused by Highs_runQuiet() who don't understand what passing NULL as a file name might mean!

Reply:
I wouldn't necessarily worry about back-compatibility at the moment. HiGHS still doesn't have an official release, so now is the time to remove/rename API that you don't want to support going forward.

Reply:
It's very tempting. There are some names that I'd like to change, and methods that I'd like to remove. 

It'll take some courage, as HiGHS is already being used in other packages, so we risk pissing some folk off. 

I suppose that if we document it in Release Notes for the official Version 1.0.0 then it's OK

Reply:
> It'll take some courage, as HiGHS is already being used in other packages, so we risk pissing some folk off.

If you don't have a released version, there can't be any compatibility guarantees! Anyone building off a specific git commit is doing at their own peril.

Who is likely to be impacted?

- Scipy are using this fork: https://github.com/mckib2/HiGHS. So they have to deal with any API changes when updating anyway.

The binaries we build in Julia are versioned (even if upstream HiGHS is not), so there won't be any issues for us.

Reply:
There are also some companies and quite a few individuals using HiGHS. I need to stop worrying about upsetting people. Requiring them to look at Release Notes allows me to plead the case for folk using HiGHS to tell us, since it's only due to such "Impact" that I'm being given so much time to work on it.

Reply:
I guess the alternative is to make a list of everything you want to change, and then immediately before tagging a 1.0 release, make all the changes, and document which methods were renamed. Then people only need to make changes once, and it's for a good cause of a 1.0 release with API compatibility guarantees.